landing-pages/site/content/en/survey/_index.html

```diff
@@ -1,22 +1,14 @@
----
-title: "Airflow Survey 2024"
-linkTitle: "Airflow Survey 2024"
-tags: ["community", "survey", "users"]
-menu:
-  main:
-    weight: 31
----
-
-<div>
-  <img src="Airflow Survey 2024 Results.png" alt="Airflow Survey 2024 Infographic" />
-</div>
-
-<div class="survey-button-container">
-  <div class="survey-button">
-    <a href="https://astronomer.typeform.com/report/SF2VGNTc/fRSeRcKKJ3kgYXVl">
-      <button type="submit" class="btn-hollow btn-brown btn-with-icon with-box-shadow">
-        <span class="bodytext__medium--brownish-grey">View Raw Data</span>
-      </button>
-    </a>
-  </div>
-</div>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="refresh" content="0; url=https://airflow.apache.org/blog/airflow-survey-2023/">
+  <link rel="canonical" href="https://airflow.apache.org/blog/airflow-survey-2023/">
+  <title>Redirecting...</title>
+</head>
+<body>
+  <p>If you are not redirected automatically, please click
+    <a href="https://airflow.apache.org/blog/airflow-survey-2023/">here</a>.
+  </p>
+</body>
+</html>
```

Comment:
why do we need this? Old 2023 link is still active ( in the form of blogpost now, instead being on the menu) right?

Reply:
where are we referring to the old link in the code snippet?

Reply:
`landing-pages/site/content/en/survey/_index.html` contains code for [https://airflow.apache.org/survey](https://airflow.apache.org/survey).

When [https://airflow.apache.org/survey](https://airflow.apache.org/survey) is loaded, it redirects to https://airflow.apache.org/blog/airflow-survey-2023/

Reply:
I have tested the redirection locally.